wnfs/README.md

```diff
@@ -7,7 +7,7 @@
 
```

Comment:
Minor links and texts fixes:
- `crates/fs/examples/` -> `examples/`
- `../../assets/logo.png` -> `../assets/logo.png`